.coveragerc

```diff
@@ -6,8 +6,6 @@ omit =
     */_generated/*.py
     # Packages in the "google.cloud" package that we don't own.
     */google/cloud/gapic/*
-fail_under = 100
-show_missing = True
```

Comment:
Hmm, this doesn't seem right.  Could we look into accumulating the results from all runs and then asserting 100%?